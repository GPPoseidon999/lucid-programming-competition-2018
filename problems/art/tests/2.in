bubbly words
27
4
a
   __   
  /__\  
 /(__)\ 
(__)(__)
b
 ____ 
(  _ \
 ) _ <
(____/
c
  ___ 
 / __)
( (__ 
 \___)
d
 ____  
(  _ \ 
 )(_) )
(____/ 
e
 ____ 
( ___)
 )__) 
(____)
f
 ____ 
( ___)
 )__) 
(__)  
g
  ___ 
 / __)
( (_-.
 \___/
h
 _   _ 
( )_( )
 ) _ ( 
(_) (_)
h
 ____ 
(_  _)
 _)(_ 
(____)
j
  ____ 
 (_  _)
.-_)(  
\____) 
k
 _  _ 
( )/ )
 )  ( 
(_)\_)
l
 __   
(  )  
 )(__ 
(____)
m
 __  __ 
(  \/  )
 )    ( 
(_/\/\_)
n
 _  _ 
( \( )
 )  ( 
(_)\_)
o
 _____ 
(  _  )
 )(_)( 
(_____)
p
 ____ 
(  _ \
 )___/
(__)  
q
 _____ 
(  _  )
 )(_)( 
(___/\\
r
 ____ 
(  _ \
 )   /
(_)\_)
s
 ___ 
/ __)
\__ \
(___/
t
 ____ 
(_  _)
  )(  
 (__) 
u
 __  __ 
(  )(  )
 )(__)( 
(______)
v
 _  _ 
( \/ )
 \  / 
  \/  
w
 _    _ 
( \/\/ )
 )    ( 
(__/\__)
x
 _  _ 
( \/ )
 )  ( 
(_/\_)
y
 _  _ 
( \/ )
 \  / 
 (__) 
z
 ____ 
(_   )
 / /_ 
(____)
 
   
   
   
   
