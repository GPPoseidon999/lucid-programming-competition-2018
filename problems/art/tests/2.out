 ____  __  __  ____  ____  __    _  _     _    _  _____  ____  ____   ___ 
(  _ \(  )(  )(  _ \(  _ \(  )  ( \/ )   ( \/\/ )(  _  )(  _ \(  _ \ / __)
 ) _ < )(__)(  ) _ < ) _ < )(__  \  /     )    (  )(_)(  )   / )(_) )\__ \
(____/(______)(____/(____/(____) (__)    (__/\__)(_____)(_)\_)(____/ (___/
